### DISCUSSION TITLE
skoala/registry/managed/registry-lcm/update-registry/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1552] [Announcements]
# skoala/registry/managed/registry-lcm/update-registry/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/skoala/registry/managed/registry-lcm/update-registry/

**Comment by liyuerich** [comment]
更新后在哪里能看到变化，或者更新的log，轨迹？
